lib/ui/addon/components/disclosure-triangle.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if @toggle}}
   <button
-    class="disclosure-triangle {{if @expanded "expanded" "collapsed"}}"
+    class="disclosure-triangle {{if @expanded "expanded" "collapsed"}} appearance-none inline-block border-0 cursor-pointer bg-transparent"
```

Comment:
What is `appearance-none`? Is that setting `display: none`? If so, what toggles it being visible and removes the class?

Reply:
`appearance-none` removes platform specific styling on things like buttons and selects.

Reply:
Ah gotcha